### DISCUSSION TITLE
Dockerfile installing versioned binaries via package manager

### DISCUSSION THREAD

**Discussion by mrupnikm** [renovatebot/renovate#29114] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab Renovate Runner pipeline

### Please tell us more about your question or problem

I am trying to understand if there is a way for Renovate to lookup the versions of tools like curl, kubectl, java, maven or other inside my dockerfile if they are installed with wget, curl or a package manager? I wanted to get the github-releases to work but I seem to be getting a conflict when setting github token for the api.github.com since I am using a private gitlab token for renovate to interact. Any help would be great! 

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
You shouldn't get any conflicts between gitlab and github tokens, if you do then you have a config mistake. If you're running on gitlab, then define a hostRules entry with matchHost=github.com and then you're github PAT.
